packages/squiggle-lang/src/ast/peggyParser.peggy

```diff
@@ -50,19 +51,26 @@ quotedInnerBlock
     '}'
     { return h.nodeBlock([finalExpression], location()); }
 
-statementsList = statement|1.., statementSeparator|
+statementsList = decoratedStatement|1.., statementSeparator|
 
-statement
-  = letStatement
-  / defunStatement
-  / voidStatement
-
-voidStatement
-  = "call" _nl value:zeroOMoreArgumentsBlockOrExpression
```

Comment:
Voids are removed in grammar (so `call ...` won't work, but AFAIK we never used it). Voids are still present in values, I'll clean that up in a separate PR, as I described in https://discord.com/channels/1130609991993274510/1133551706370752582/1186439261763948584.